src/app/components/UserErrorComponent.js

```diff
@@ -43,6 +44,8 @@ const UserErrorComponent = ({
         return strings.could_not_encode_address;
       case TRANSACTION_RECEIPT_FAILED:
         return strings.transaction_receipt_failed;
+      case ERROR_NOT_ENOUGH_RIF:
```

Comment:
NIT: @annipi I know you didn't introduce it, but what do you think if we refactor this with something like:
```
const errorMessages = {
    ERROR_SAME_VALUE: strings.same_value,
    ERROR_RESOLVE_NAME: strings.resolve_not_set,
    ADDRESS_ENCODING_ERROR: strings.could_not_encode_address,
    TRANSACTION_RECEIPT_FAILED: strings.transaction_receipt_failed,
    ERROR_NOT_ENOUGH_RIF: strings.not_enough_rif_balance,
  };

  const formatMessage = () => errorMessages[message] || truncateErrorMessage(message);
```

Reply:
Ok, done!

Reply:
@antomor  I didn't initially understand the change, now i t it fixed, I removed the switch case and make `errorMessages` an object.